.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -5,10 +5,13 @@ about: Create a report to help us improve
 ---
 
 <!--
+====================================
+IF YOUR ISSUE IS RELATED TO SECURITY
+====================================
+please submit it to the security mailing-list security@riot-os.org.
+
 If your issue is a usage question, please submit it to the user mailing-list
```

Comment:
is related to the usage of RIOT, ...

Reply:
Not really part of this PR, but will amend in a separate commit.

Reply:
:exclamation:  Does this mean that any  security related issue will be immediately sent to all subscribers? Unless those subscribers is a very closed group, that is probably not a good idea.